### PR TITLE
Add matrix and remove double bootstrapping from insomnia-components

### PR DESCRIPTION
**Pull request by develohpanda**

### CODE REVIEW

File: packages/insomnia-components/package.json [hunk 1]
```diff
@@ -13,9 +13,8 @@
     "watch": "webpack --config webpack/webpack.dev.js --watch",
     "build-storybook": "build-storybook --quiet",
     "storybook": "start-storybook --port 3223",
-    "bootstrap": "npm run convert-svg && npm run build",
     "convert-svg": "rimraf assets/svgr && svgr --out-dir assets/svgr assets && rimraf assets/svgr/index.js",
-    "prepublish": "npm run convert-svg && npm run build"
+    "prepack": "npm run convert-svg && npm run build"
```
Comment (develohpanda) [inline packages/insomnia-components/package.json:17]: FYI [prepublish](https://docs.npmjs.com/misc/scripts#deprecation-note) has been deprecated in favour of prepare

Comment (develohpanda) [inline packages/insomnia-components/package.json:17]: FYI [prepublish](https://docs.npmjs.com/misc/scripts#deprecation-note) has been deprecated in favour of prepare

File: packages/insomnia-components/package.json [hunk 2]
```diff
@@ -13,9 +13,8 @@
     "watch": "webpack --config webpack/webpack.dev.js --watch",
     "build-storybook": "build-storybook --quiet",
     "storybook": "start-storybook --port 3223",
-    "bootstrap": "npm run convert-svg && npm run build",
     "convert-svg": "rimraf assets/svgr && svgr --out-dir assets/svgr assets && rimraf assets/svgr/index.js",
-    "prepublish": "npm run convert-svg && npm run build"
+    "prepare": "npm run convert-svg && npm run build"
```
Comment (develohpanda) [inline packages/insomnia-components/package.json:17]: FYI [prepublish](https://docs.npmjs.com/misc/scripts#deprecation-note) has been deprecated in favour of prepare

Comment (nijikokun) [reply packages/insomnia-components/package.json:17]: > As of `npm@4.0.0`, a new event has been introduced, prepare, that preserves this existing behavior.

This does mean that we now *require* `npm >= v4.0.0` to dev, as a note.

**Review by gschier — APPROVED** [review]
Nice little simplification 😄

**Review by DMarby — APPROVED** [review]
🎉

**Review by sonicyeti — APPROVED** [review]
Sweet!

**Pull request merged**